### PR TITLE
1706-BE-Sam N

### PR DESCRIPTION
**Pull request by snayrouz**

### CODE REVIEW

File: test/activity_test.rb [hunk 1]
```diff
@@ -4,4 +4,79 @@
...
+    assert_equal expected , actual
+  end
+
+  def test_can_add_to_participants
+    a = Activity.new("Brunch")
+
+    actual = a.add_participant
+    expected = {"Jim" => 20}
+
+    assert_equal expected , actual
```
Comment (s-espinosa) [inline test/activity_test.rb]: Careful here: interaction pattern indicates that `a.participants` will return `{"Jim" => 20}`. You're currently asserting that `a.add_participant` will return that. Also, you're not passing `#add_participant` an argument, which is needed per the spec.

File: test/activity_test.rb [hunk 2]
```diff
@@ -4,4 +4,79 @@
...
+  end
+
+  def test_can_show_owed_amount_by_person
+    a = Activity.new("Brunch")
+
+    actual = a.owed
+    expected = {"Jim" => 10, "Joe" => -10}
+
+    assert_equal expected, actual
+  end
```
Comment (s-espinosa) [inline test/activity_test.rb]: Rest of these tests look good.

File: test/reunion_test.rb [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
+    r = Reunion.new("Denver")
+
+    assert_equal "Denver", r.name
+  end
+
+  def test_activities_starts_empty
+    r = Reunion.new("Denver")
+
+    assert_equal [], r.activities
+  end
```
Comment (s-espinosa) [inline test/reunion_test.rb]: Also like these tests.

**Comment by snayrouz** [comment]
I can confidently say I was off this morning. I knew I wasn't doing well so I ran through to set up tests to see if I could start from there

**Pull request closed without merging**